nl/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
-password = Enter your master password.
+firstPassword = Een master wachtwoord wordt gebruikt om gevoelige informatie met eenmalige aanmelding te beschermen die is opgeslagen op uw hard drive.  Toekomstige releases zullen deze gegevens sterk vergrendelen voor maximale bescherming.##Selecteer een master wachtwoord.#(Laat de velden leeg als u geen master wachtwoord wilt gebruiken).#
+password = Voer uw master wachtwoord in.
 #translator's note: # in following line will be converted to linefeed by program
-newPassword = Select a new master password.#(Leave fields blank if you don't want to use a master password).#
-enterPassword = Enter a master password
-confirmPassword = Retype the master password
-confirmFailed_TryAgain? = The two master passwords were not identical.  Please enter the same master password in both fields.
-SelectUser = Select a username to be entered on this form.
-SelectUserWhosePasswordIsBeingChanged = Select the user whose password is being changed.
-PasswordNotification = For your convenience, the browser can remember your user names and passwords so you won't have to re-type them in the future.  Your passwords will be obscured before being saved on your hard drive.  Do you want this feature enabled?
-WalletNotification = You can save information that you enter on forms and later automatically prefill that information on other forms.  To save such information, select Edit/Save Form Info from the menu while viewing the form.
-noPrefills = There are no fields that can be prefilled.
-Never = Never for this site
-Confirm = Confirm
-PromptForPassword = Password
+newPassword = Selecteer een nieuw master wachtwoord.#(Laat de velden leeg als u geen master wachtwoord wilt gebruiken).#
+enterPassword = Voer een master wachtwoord in
+confirmPassword = Typ het master wachtwoord opnieuw
+confirmFailed_TryAgain? = De twee master wachtwoorden waren niet identiek.  Voor alstublieft in beide velden hetzelfde master wachtwoord in.
+SelectUser = Selecteer een gebruikersnaam om in te voeren in dit formulier.
+SelectUserWhosePasswordIsBeingChanged = Selecteer de gebruiker wiens wachtwoord wordt gewijzigd.
+PasswordNotification = Voor uw gemak kan de browser uw gebruikersnamen en wachtwoorden onthouden zodat u ze niet opnieuw hoeft in te typen in de toekomst.  Uw wachtwoorden worden verborgen voordat ze worden opgeslagen op uw hard drive.  Wilt u dat deze functie ingeschakeld is?
+WalletNotification = U kunt informatie die u invoert in formulieren opslaan en later deze informatie automatisch laten invullen op andere formulieren.  Selecteer om zulke informatie op te slaan de Bewerken/Opslaan Formulier Informatie van het menu terwijl u het formulier bekijkt.
+noPrefills = Er zijn geen velden die voorgevuld kunnen worden.
```

Comment:
CORRECTION NEEDED, it should be: ''Er zijn geen velden die vooraf ingevuld kunnen worden.''